base_geoengine/static/src/css/style.css

```diff
@@ -0,0 +1,402 @@
+/* OpenLayers Style */
+:root,
+:host {
+    --ol-background-color: white;
+    --ol-accent-background-color: #f5f5f5;
+    --ol-subtle-background-color: rgba(128, 128, 128, 0.25);
+    --ol-partial-background-color: rgba(255, 255, 255, 0.75);
+    --ol-foreground-color: #333333;
+    --ol-subtle-foreground-color: #666666;
+    --ol-brand-color: #00aaff;
+}
+
+.ol-box {
+    box-sizing: border-box;
+    border-radius: 2px;
+    border: 1.5px solid var(--ol-background-color);
+    background-color: var(--ol-partial-background-color);
+}
+
+.ol-mouse-position {
+    top: 8px;
+    right: 8px;
+    position: absolute;
+}
+
+.ol-scale-line {
+    background: var(--ol-partial-background-color);
+    border-radius: 4px;
+    bottom: 8px;
+    left: 8px;
+    padding: 2px;
+    position: absolute;
+}
+
+.ol-scale-line-inner {
+    border: 1px solid var(--ol-subtle-foreground-color);
+    border-top: none;
+    color: var(--ol-foreground-color);
+    font-size: 10px;
+    text-align: center;
+    margin: 1px;
+    will-change: contents, width;
+    transition: all 0.25s;
+}
+
+.ol-scale-bar {
+    position: absolute;
+    bottom: 8px;
+    left: 8px;
+}
+
+.ol-scale-bar-inner {
+    display: flex;
```

Comment:
the guideline in odoo is now to prefer using boostrap css class above custom css. It helped a lot to reduce repetitive common css properties, such as display flex. So, for example, here, you could simply set `d-flex` on the element. (note that it may not work here, if you don't own the code that render this css class). Just saying that if you want, you can probably replace some of your css with bootstrap.